src/Ace/Internal.purs

```diff
@@ -0,0 +1,19 @@
+module Ace.Internal 
```

Comment:
Would this be a good candidate for a foreign lib (purescript-nullable or something)?